addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4222,7 +4255,12 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             } else {
                 shapeToSelect = possibleShapes[1];
             }
-            return this._handlePreviewState(false, () => ({shape: shapeToSelect}));
+            _.defer(() => {
+                // options for shape will only be available after _toggleShape() returned
+                const select = $("we-select.o_we_shape_menu we-toggler");
+                select.trigger('click');
```

Comment:
There is a method to find a widget by name (something like `requestUserValueWidget`) and a method to simulate a click on it. Don't search through the whole DOM, you could actually find another shape menu.